keyboards/hyper7/h7/v3/keymaps/default/keymap.c

```diff
@@ -0,0 +1,760 @@
+#include QMK_KEYBOARD_H
+
+#include "../../config.h"
+#include "../../../../../../quantum/quantum.h"
+#include "../../../../../../quantum/quantum_keycodes.h"
+
```

Comment:
```suggestion

```

Reply:
These were added to help with local development - happy to remove, but could someone point me in the direction of some documentation showing how to set up a local env?